esp/src/src/ESPUtil.ts

```diff
@@ -249,6 +251,16 @@ export var IdleWatcher = dojo.declare([Evented], {
     constructor: function (idleDuration) {
         idleDuration = idleDuration || 30 * 1000;
         this._idleDuration = idleDuration;
+
+        var context = this;
+        var prevLocked;
+        this._intervalLockHandle = setInterval(function () {
+            var locked = cookie("Status") === "locked";
+            if (!locked && prevLocked) {
```

Comment:
This keeps running over and over. Should probably only run when idle? Looking into